models/topicsModel.js

```diff
@@ -0,0 +1,10 @@
+const db = require('../db/connection'); 
+const fetchTopics = () => {
+    return db
+    .query("SELECT * FROM topics;")
+    .then(({rows}) => { //forgot why {rows? ask tutor}
```

Comment:
With .then we can name the argument of the callback function whatever we want ( I will normally do .then((result) => {} )

When the .then is following a db.query with SQL then whatever we name this, it will be an object which will contain a key of rows. The value of rows will be an array containing the results of the SQL query so in this case it will be an array of topic objects.

{rows} is destructuring the response from db.query and takes the rows property itself. {rows} is the same as result.rows :)

Reply:
aaah Thank you! That makes sense :)